logic/standalone/aligner.py

```diff
@@ -0,0 +1,131 @@
+'''
+Aligner scans the whole level, then process all objects with name as "water_line".
+
+
+USAGE EXAMPLE:
+    aligner.AlignWater(playdo)
+'''
+
+import logic.common.log_utils as log
+import logic.common.tiled_utils as tiled_utils
+
+#--------------------------------------------------#
+'''Variables'''
+
+# For logging
+CHAR_TRUE  = "  ○ "
+CHAR_FALSE = "  ☓ "
+
+
+
+#--------------------------------------------------#
+'''Public Functions'''
+
+def AlignWater(playdo):
+    log.Info("\n--- Aligning water_line Objects ---\n")
+
+    # Get objectgroup that contains the water
+    list_of_all_water_lines = playdo.GetAllObjectsWithName("water_line")
+    log.Must(f'Found {len(list_of_all_water_lines)} waterlines')
+
+    # Process individual object
+    count_need_alignment = 0
+    log.Info("--------------------------------------------------")
+    for count, water_line_obj in enumerate(list_of_all_water_lines):
```

Comment:
for loops, the name "index" instead of "count" is preferred.

Plus, you're using the word "count" for "count_need_alignment", so it'd help to add more separation between the two